schutzbot/scheduled_cloud_cleaner.sh

```diff
@@ -84,3 +84,114 @@ for i in $(seq 0 $(("$STORAGE_ACCOUNT_COUNT"-1))); do
 done
 
 echo "Azure cleanup complete!"
+
+
+#---------------------------------------------------------------
+# 			AWS cleanup
+#---------------------------------------------------------------
+
+greenprint "Starting aws cleanup"
+
+TEMPDIR=$(mktemp -d)
+function cleanup() {
+    sudo rm -rf "$TEMPDIR"
+}
+trap cleanup EXIT
+
+# Check available container runtime
+if which podman 2>/dev/null >&2; then
+    CONTAINER_RUNTIME=podman
+elif which docker 2>/dev/null >&2; then
+    CONTAINER_RUNTIME=docker
+else
+    echo No container runtime found, install podman or docker.
+    exit 2
+fi
+
+CONTAINER_IMAGE_CLOUD_TOOLS="quay.io/osbuild/cloud-tools:latest"
+
+SSH_DATA_DIR=$(/usr/libexec/osbuild-composer-test/gen-ssh.sh)
+#SSH_KEY=${SSH_DATA_DIR}/id_rsa
+
+# We need awscli to talk to AWS.
+if ! hash aws; then
+    echo "Using 'awscli' from a container"
+    sudo ${CONTAINER_RUNTIME} pull ${CONTAINER_IMAGE_CLOUD_TOOLS}
+
+#----------------------------------
+#TODO: Do we need all this options?
+#----------------------------------
```

Comment:
I suppose we do, this gives the container the necessary env vars and also mounts required directories inside it.